tests/localosmosis/state_export/scripts/start.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+set -e 
+set -o pipefail
+
+OSMOSIS_HOME=$HOME/.osmosisd
+CONFIG_FOLDER=$OSMOSIS_HOME/config
+
+DEFAULT_MNEMONIC="bottom loan skill merry east cradle onion journey palm apology verb edit desert impose absurd oil bubble sweet glove shallow size build burst effort"
+DEFAULT_CHAIN_ID="localosmosis"
+DEFAULT_MONIKER="val"
+
+# Override default values with environment variables
+MNEMONIC=${MNEMONIC:-$DEFAULT_MNEMONIC}
+CHAIN_ID=${CHAIN_ID:-$DEFAULT_CHAIN_ID}
+MONIKER=${MONIKER:-$DEFAULT_MONIKER}
+
+install_prerequisites () {
+    apk add -q --no-cache \
+        dasel \
+        python3 \
+        py3-pip
+}
+
+edit_config () {
+
+    # Remove seeds
+    dasel put string -f $CONFIG_FOLDER/config.toml '.p2p.seeds' ''
+
+    # Disable fast_sync
+    dasel put bool -f $CONFIG_FOLDER/config.toml '.fast_sync' 'false'
+
+    # Expose the rpc
+    dasel put string -f $CONFIG_FOLDER/config.toml '.rpc.laddr' "tcp://0.0.0.0:26657"
+}
+
+if [[ ! -d $CONFIG_FOLDER ]]
+then
+
+    install_prerequisites
+
+    echo "Chain ID: $CHAIN_ID"
+    echo "Moniker:  $MONIKER"
+
```

Comment:
should we start with an unsafe-reset-all? my main minor concern is if they have a filled addrbook, this will carry over and they will have a ton of p2p logs trying to call (probably mainnet) IPs

Reply:
Yeah, it has some verbose p2p logs at the beginning.
Let's test it and see if it's better

Reply:
For sure is not a big deal though and we can skip this issue for later! I'm going to mark resolved unless you feel it needs further exploration